conda_lock/conda_solver.py

```diff
@@ -492,7 +492,14 @@ def update_specs_for_arch(
                     f"Could not lock the environment for platform {platform}: {err_json.get('message')}"
                 ) from exc
 
-            dryrun_install: DryRunInstall = json.loads(proc.stdout)
+            conda_update_output = proc.stdout
+
+            # <https://github.com/conda/conda-lock/pull/448#issuecomment-1627530087>
+            conda_update_output = conda_update_output.replace(
+                "No package record found!", ""
+            )
+
+            dryrun_install: DryRunInstall = json.loads(conda_update_output)
```

Comment:
@maresb, I looked a bit around and there is maybe something cleaner for the `No package record found!` thing, using something `extract_json_object` that is already in place for other `--json --dry-run` outputs e.g. in

https://github.com/conda/conda-lock/blob/5bbb1a6b30202b2c864a80cfc4532129b1933568/conda_lock/conda_solver.py#L392C1-L392C85

```suggestion
            dryrun_install: DryRunInstall = json.loads(extract_json_object(proc.stdout))
```

Tested locally: it does its job like charm!